src/edu/washington/escience/myria/parallel/Server.java

```diff
@@ -1099,52 +1101,63 @@ public long createView(
   /**
    * Create a function and register it in the catalog
    */
-  public long createFunction(
-      final String functionName, final String functionDefinition, final Set<Integer> workers)
+  public String createFunction(
+      final String functionName,
+      final String functionDefinition,
+      final String functionOutputSchema,
+      final Set<Integer> workers)
       throws DbException, InterruptedException {
-    long queryID;
+    String response = "Created Function";
     Set<Integer> actualWorkers = workers;
     if (workers == null) {
       actualWorkers = getWorkers().keySet();
     }
 
-    /* Create the function */
-    try {
-      Map<Integer, SubQueryPlan> workerPlans = new HashMap<>();
-      for (Integer workerId : actualWorkers) {
-        workerPlans.put(
-            workerId,
-            new SubQueryPlan(
-                new DbCreateFunction(
-                    EmptyRelation.of(Schema.EMPTY_SCHEMA),
-                    functionName,
-                    functionDefinition,
-                    null)));
-      }
-      ListenableFuture<Query> qf =
-          queryManager.submitQuery(
-              "create function",
-              "create function",
-              "create function",
-              new SubQueryPlan(new SinkRoot(new EOSSource())),
-              workerPlans);
+    /* Validate the command */
+    Pattern pattern = Pattern.compile("(CREATE FUNCTION)([\\s\\S]*)(LANGUAGE SQL;)");
```

Comment:
Here and below we are getting into Postgres-specific syntax, so it might be good to call that out at least in comments. The "portable" approach, of course,  would be to add a `createFunction()` method to the `AccessMethod` interface; did we discuss that option?